### PR TITLE
add getOrganization, getRepos, getRepo, createRepo, deleteRepo method…

### PR DESCRIPTION
**Pull request by andrewayoub**
add getOrganizations, getRepos, getRepo, createRepo and deleteRepo methods to Githab client

#### What this PR resolves:
adds getOrganizations, getRepos, getRepo, createRepo and deleteRepo methods to Githab client

#### Changes proposed in this PR:

- add getOrganizations
- add getRepos
- add getRepo
- add createRepo
- add deleteRepo

**Version**: 

**Fixes**: #138

### CODE REVIEW

**Comment by yveskerwyn** [comment]
@rkhamis, can we use the [PEP 8](https://www.python.org/dev/peps/pep-0008/) naming convention for these new methods?

get_organizations() instead of getOrganizations()

**Comment by rkhamis** [comment]
@andrewayoub @yveskerwyn good idea... let's make it object first than action (because that's how we'll refactor)

so organization_get, repo_get, repo_delete, repo_create....



also @andrewayoub please add tests

File: JumpScale9Lib/clients/github/Github.py [hunk 1]
```diff
@@ -61,3 +62,72 @@ def getUser(self, login="", githubObj=None):
...
+        """
+        return self.api.get_user().get_orgs()
+
+    def repos_get(self, organizationId=None):
+        """
+        gets all repos for a user if organizationId=None otherwise it return only repos for this organization
+
+        Args:
+            - organizationId (optional): Id for the organization to get repos from
+
```
Comment (ahussein) [inline JumpScale9Lib/clients/github/Github.py]: can we stick to the same convention when adding docstrings for params, normally we use:
@param <paramname>: <description>

File: JumpScale9Lib/clients/github/Github.py [hunk 2]
```diff
@@ -61,3 +62,72 @@ def getUser(self, login="", githubObj=None):
...
+        Args:
+            - name (required): repo name
+            - description (optional): repo desription
+            - homepage (optional): the home page for the repo
+            - private (optional): if true the repo will be created as private repo
+            - has_issues (optional): indicates that the repo has issues or no
+            - has_wiki (optional): indicates that the repo has wiki or no
+            - has_downloads (optional): indicates that the repo has downloads or no
+            - auto_init (optional): if true the repo will be automaticly initialized
+            - gitignore_template (optional): the gitignore template
```
Comment (ahussein) [inline JumpScale9Lib/clients/github/Github.py]: same as above for docstrings

File: JumpScale9Lib/clients/github/Github.py [hunk 3]
```diff
@@ -61,3 +62,72 @@ def getUser(self, login="", githubObj=None):
...
+
+    def repo_delete(self, repo):
+        """
+        deletes a repo
+
+        Args:
+            - repo (required): a repo to be deleted
+
+        Raises:
+            - RuntimeError if repo is not a valid Repository
```
Comment (ahussein) [inline JumpScale9Lib/clients/github/Github.py]: same as above for docstring

File: JumpScale9Lib/clients/github/test_Github.py [hunk 4]
```diff
@@ -0,0 +1,106 @@
+import pytest
+from js9 import j
+
+from .Github import *
```
Comment (ahussein) [inline JumpScale9Lib/clients/github/test_Github.py]: normally, its not a very good practice to import *, please import the attributes that you need.

**Comment by andrewayoub** [comment]
@ahussein  please review the changes

**Comment by andrewayoub** [comment]
moved to https://github.com/Jumpscale/lib9/pull/163

**Pull request closed without merging**